Unite.Web/Client/src/admin/components/SubmissionsTab.vue

```diff
@@ -0,0 +1,218 @@
+
+  <div class="col">
+    <div class="row q-col-gutter-xs">
+      <!-- Left -->
+      <div class="col-3">
+        <q-table
+          class="u-sticky-header-admin"
+          separator="cell"
+          row-key="id"
+          :rows="rows"
+          :columns="columns"
+          :pagination="{ rowsPerPage: 20 }"
+          :loading="loadingSubmissions"
+          @row-click="onRowSelected"
+          dense flat bordered
+        >
+
+        <template v-slot:body-cell="props">
+          <q-td :props="props" :class="{ 'text-primary': props.row.id == row?.id }">
+            {{ props.value }}
+          </q-td>
+        </template>
+        </q-table>
+      </div>
+
+      <!-- Right -->
+      <div class="col-9" v-if="submission">
+        <q-card flat bordered>
+          <div class="row q-py-xs q-px-md justify-between items-center">
+            <div>
+              Id: <b>{{ row?.id }}</b>, Type: <b>{{ getTypeLabel(row?.type) }}</b>, Date: <b>{{ getDateLabel(row?.date) }}</b> 
+            </div>
+            <div>
+              <q-btn label="Approve" color="green" icon="las la-chevron-circle-down" dense flat no-caps @click="onApproveSubmission"/>
+              <q-btn label="Reject" color="red" icon="las la-times-circle" dense flat no-caps @click="onRejectSubmission"/>
+            </div>
+          </div>
+
+          <q-separator />
+
+          <div class="row q-py-xs q-px-md">
+            <div class="col" style="height: 480px; overflow: auto;">
+              <div v-if="isComplexType(row?.type)">  
+                <u-json-viewer :value="submission"/>
+              </div>
+              <div v-else>
+                <u-tsv-viewer :value="submission"/>
+              </div>
+            </div>
+          </div>
+        </q-card>
+      </div>
+      <div class="col-9" v-else>
+        <div class="q-mt-lg text-center">No submission selected</div>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+import api from "../api/api-submissions";
+import URejectDialog from "../components/submissions/RejectDialog.vue";
+
+import DonorsApi from "@/domain/donors/api";
+import ImagesApi from "@/domain/images/_shared/images/api";
+import SpecimensApi from "@/domain/specimens/_shared/specimens/api";
+import GenesApi from "@/domain/genome/genes/api";
+import VariantsApi from "@/domain/genome/variants/_shared/variants/api";
+import DonorsSubmissionType from "@/domain/donors/models/enums/submission-type";
+import ImagesSubmissionType from "@/domain/images/_shared/images/models/enums/submission-type";
+import SpecimensSubmissionType from "@/domain/specimens/_shared/specimens/models/enums/submission-type";
+import GenomesSubmissionType from "@/domain/genome/genes/models/enums/submission-type";
+import VariantsSubmissionType from "@/domain/genome/variants/_shared/variants/models/enums/submission-type";
+import UTsvViewer from '../components/submissions/TsvViewer.vue';
```

Comment:
Could you move components imports to the top section, right after URejectDialog.
Component imports are usually on the top.

And also api import in opposite should go below.